tests/boardfarm_plugins/README.md

```diff
@@ -0,0 +1,174 @@
+<!--
+SPDX-License-Identifier: BSD-2-Clause-Patent
+Copyright (c) 2020 the prplMesh contributors
+This code is subject to the terms of the BSD+Patent license.
+See LICENSE file for more details.
+-->
+
+### Prerequisites
+
+To obtain prplMesh and boardfarm sources a Google repo tool is used.
+Refer to its README for detailed info:
+https://gerrit.googlesource.com/git-repo/+/refs/heads/master/README.md
+
+### Boardfarm installation
+
+1. To obtain sources using previously mentioned repo tool run:
+```
+repo init -u https://github.com/prplfoundation/prplMesh-manifest.git
+repo sync
+repo forall -p -c 'git checkout $REPO_RREV'
+```
+
+2. Install boardfarm to your host:
+```
+sudo python3 setup.py install
```

Comment:
1. You need to add a 'cd boardfarm` before this commant
2. Is there a way to install it without sudo?

Reply:
> 1. You need to add a 'cd boardfarm` before this commant
> 2. Is there a way to install it without sudo?

1. Indeed
2. No, because we need permissions to write in `/usr/local/lib/python3.6/dist-packages` where boardfarm will be installed.

Reply:
Actually, you *can* install without sudo, you just need an additional option `--user`:
```suggestion
python3 setup.py install --user
```